packages/py3_brotlicffi.rb

```diff
@@ -0,0 +1,24 @@
+require 'package'
+
+class Py3_brotlicffi < Package
+  description 'Python bindings to the Brotli compression library'
+  homepage 'https://github.com/python-hyper/brotlicffi/'
+  @_ver = '1.0.9.2'
+  version @_ver
+  license 'MIT'
+  compatibility 'all'
+  source_url 'https://github.com/python-hyper/brotlicffi.git'
+  git_hashtag "v#{@_ver}"
+
```

Comment:
Do we need binaries or does this build fairly quickly?

Reply:
binaries, it invokes a c compiler

Reply:
installs quickly.